erpnext/assets/doctype/location/location_tree.js

```diff
@@ -27,6 +27,44 @@ frappe.treeview_settings["Location"] = {
 			condition: 'frappe.boot.user.can_create.indexOf("Location") !== -1'
 		}
 	],
+	onrender: function (node) {
+
+		if (!node.is_root) {
+			frappe.db.get_value("Location", node.data.value, "area")
+				.then((r) => {
+					// Place the location area on the right side, tree accounts style.
+					$('<span class="balance-area pull-right text-muted small">'
+						+ (`${((flt(r.message.area)).toLocaleString('en') + ' Square Meters')}`)
```

Comment:
Strings must be translatable

```js
_('{0} Sq m', flt(r.message.area).toLocaleString('en'));
```

(us abbreviation for narrow views)

Reply:
Understood, I've applied the changes to show **Sq m** in 175838e